### DISCUSSION TITLE
libkeybinder & flatpak

### DISCUSSION THREAD

**Discussion by LucasXu0** [KRTirtho/spotube#262] [Q&A] [answered]
Hi, @KRTirtho. I saw on StackOverflow that you asked keybinder-related questions. I'm currently having the same problem integrating flatpak. I would like to ask you how you solved it at that time. Thanks.

**Comment by KRTirtho** [comment, marked as answer]
See this for clear idea https://github.com/flathub/com.github.KRTirtho.Spotube/blob/ba0ac70d9810e98b92ef59512f85408792f88f19/com.github.KRTirtho.Spotube.yml#L15

**Reply by LucasXu0** [reply]
Awesome. Thanks.

**Reply by KRTirtho** [reply]
No worries. You're doing an awesome job with AppFlowy btw :)
